### ISSUE TITLE
Upload and Download module - very small size field's width upon uploading

### ISSUE THREAD

**Issue by sz00gun** [qooob/authentic-theme#320]
Not sure if this is related to either Webmin or Filemin or Authentic theme...

but this is not right, the window should be widther

![ofwhat](https://cloud.githubusercontent.com/assets/7549566/12098425/2a8a0960-b31a-11e5-8d51-d4fbc172b82a.png)

**Comment by qooob** [comment]
Which module is that?

**Comment by qooob** [comment]
Yes, understood. This can be fixed using theme. It will be fixed in the next release.

Meanwhile, if you manually open theme's _package.min.css_ and add at the very end the following, it'll work right now. Don't forget to clear cache afterwards.

```CSS
#popup > form center input[name="size"] {
    min-width: 178px;
}

```

**Comment by sz00gun** [comment]
That fix working fine, however (just noticed) the length of the progress box is wrong,

eg. 57.51 MB of 58.09 MB is around 99%, so the progress indicators should be almost on the end, but there are around the middle.
![suwak](https://cloud.githubusercontent.com/assets/7549566/14227306/5cf73a12-f8f2-11e5-912f-cb985f37a4d0.png)

**Comment by qooob** [comment]
I see.. :)

**Comment by qooob** [comment]
Will be fixed in 18.01.
